tools/ci_build/github/azure-pipelines/mac-ios-ci-pipeline.yml

```diff
@@ -0,0 +1,10 @@
+jobs:
+- job: iOS_CI_on_Mac
+  pool:
+    vmImage: 'macOS-10.15'
+  timeoutInMinutes: 120
+  steps:
+    - script: |
+        sdkpath=`xcrun --sdk iphonesimulator --show-sdk-path`
+        python3 $(Build.SourcesDirectory)/tools/ci_build/build.py --build_dir iOS --ios --ios_sysroot $sdkpath --osx_arch x86_64 --apple_deploy_target 12.1 --use_xcode --config RelWithDebInfo
```

Comment:
>--ios_sysroot $sdkpath [](start = 88, length = 22)

Curious, '--ios_sysroot iphonesimulator' doesn't work here? Do you have to use a full path to the sdk?
Also, use --config Debug?
And add --parallel?

Reply:
have you ever succeeded with this short name instead of the full path? I didn't see how it works.

The azure machine pool is still slow, it will easily hit time-out if adding debug, which is also slow and I never use this flavor.
let's try whether --parallel can make it faster.

Reply:
For me, I always use  '--ios_sysroot iphonesimulator' on my Mac, I think it should work on azure mac machine
Azure Mac pipeline machine is a hit or miss, for Android CI, sometimes it can finish within 30 minutes, but sometimes can take 1.5 hours, maybe change timeout to 150 minutes?

Reply:
The short name trick actually is done by cmake and I did get the issue before. So I always use the explicit path just like xcode did and it looks more robust.

Reply:
@gwang-msft ,  the result of '--parallel' is a little bit slower than single process, according to the pipeline log, 45m vs 40m. It looks the azure machine only provide 1 physical core for the job.